Gemfile

```diff
@@ -73,7 +73,7 @@ group :development, :test do
   gem 'byebug'
   gem 'database_cleaner'
   gem 'dotenv'
-  gem 'html-proofer'
+  gem 'html-proofer', '< 4'
```

Comment:
Skipped in https://github.com/glowfic-constellation/glowfic/pull/1775#issuecomment-1732374681, since 4.x and 5.x seem to stop doing strict HTML parsing